core/lib.rs

```diff
@@ -355,7 +361,10 @@ impl Connection {
                 Cmd::ExplainQueryPlan(_stmt) => todo!(),
             }
         } else {
-            todo!()
+            // Shouln't happen
+            Err(LimboError::ParseError(
```

Comment:
if we consider this unreachable, then should it be `let cmd = cmd.expect("successfuly parse on nonempty input string should produce a command");`